evm/vm/flavors/tester/__init__.py

```diff
@@ -0,0 +1,124 @@
+from cytoolz import (
+    assoc,
+)
+
+from eth_utils import (
+    reversed_return,
+)
+
+from evm import (
+    Chain,
+)
+
+from evm.vm.flavors import (
+    FrontierVM as BaseFrontierVM,
+    HomesteadVM as BaseHomesteadVM,
+    EIP150VM as BaseEIP150VM,
+)
+
+from evm.utils.chain import (
+    generate_vms_by_range,
+)
+
+
+class MaintainGasLimitMixin(object):
+    @classmethod
+    def create_tester_header_from_parent(cls, parent_header, **header_params):
```

Comment:
Shouldn't this be `create_header_from_parent`?

Reply:
YES! good catch.